src/chocolatey.resources/chocolatey.resources.csproj

```diff
@@ -64,98 +64,206 @@
     <Compile Include="Properties\AssemblyInfo.cs" />
   </ItemGroup>
   <ItemGroup>
-    <EmbeddedResource Include="helpers\chocolateyInstaller.psm1" />
-    <EmbeddedResource Include="helpers\functions\Format-FileSize.ps1" />
-    <EmbeddedResource Include="helpers\functions\Get-CheckSumValid.ps1" />
-    <EmbeddedResource Include="helpers\functions\Get-ChocolateyConfigValue.ps1" />
-    <EmbeddedResource Include="helpers\functions\Get-ChocolateyPath.ps1" />
-    <EmbeddedResource Include="helpers\functions\Get-ChocolateyUnzip.ps1" />
-    <EmbeddedResource Include="helpers\functions\Get-ChocolateyWebFile.ps1" />
-    <EmbeddedResource Include="helpers\functions\Get-FtpFile.ps1" />
-    <EmbeddedResource Include="helpers\functions\Get-OSArchitectureWidth.ps1" />
-    <EmbeddedResource Include="helpers\functions\Get-UACEnabled.ps1" />
-    <EmbeddedResource Include="helpers\functions\Get-VirusCheckValid.ps1" />
-    <EmbeddedResource Include="helpers\functions\Get-WebFile.ps1" />
-    <EmbeddedResource Include="helpers\functions\Get-WebHeaders.ps1" />
-    <EmbeddedResource Include="helpers\functions\Install-ChocolateyEnvironmentVariable.ps1" />
-    <EmbeddedResource Include="helpers\functions\Install-ChocolateyExplorerMenuItem.ps1" />
-    <EmbeddedResource Include="helpers\functions\Install-ChocolateyFileAssociation.ps1" />
-    <EmbeddedResource Include="helpers\functions\Install-ChocolateyInstallPackage.ps1" />
-    <EmbeddedResource Include="helpers\functions\Install-ChocolateyPackage.ps1" />
-    <EmbeddedResource Include="helpers\functions\Install-ChocolateyPinnedTaskBarItem.ps1" />
-    <EmbeddedResource Include="helpers\functions\Install-ChocolateyPowershellCommand.ps1" />
-    <EmbeddedResource Include="helpers\functions\Install-ChocolateyShortcut.ps1" />
-    <EmbeddedResource Include="helpers\functions\Install-ChocolateyVsixPackage.ps1" />
-    <EmbeddedResource Include="helpers\functions\Install-ChocolateyZipPackage.ps1" />
-    <EmbeddedResource Include="helpers\functions\Start-ChocolateyProcessAsAdmin.ps1" />
-    <EmbeddedResource Include="helpers\functions\Uninstall-ChocolateyPackage.ps1" />
-    <EmbeddedResource Include="helpers\functions\UnInstall-ChocolateyZipPackage.ps1" />
-    <EmbeddedResource Include="redirects\choco.exe.ignore" />
-    <EmbeddedResource Include="tools\checksum.exe.ignore" />
-    <EmbeddedResource Include="tools\shimgen.exe.ignore" />
+    <EmbeddedResource Include="helpers\chocolateyInstaller.psm1">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>

```

Comment:
@vexx32 why was this type of change necessary? Did you notice something during putting this PR together that meant that this was necessary?

If so, I think we need to capture this information into the commit message, for our future selves.

Reply:
Added some words to the commit message to make this clear.